ROX-18907: Updates for RHACS CS GA (#67729)
openshift/openshift-docs: cloud_service/rhacs-cloud-service-service-description.adoc

For more information, see the following resources:

* link:https://access.redhat.com/security_testing_sdl_practice[Red{nbsp}Hat Software Development Life Cycle practices]
* link:https://www.redhat.com/en/blog/security-design-security-principles-and-threat-modeling[Security by design: Security principles and threat modeling]

Applications are scanned regularly and the container scan results of the product are available publicly. For example, on the Red{nbsp}Hat Ecosystem Catalog site, you can select a component image such as `rhacs-main` and click the *Security* tab to see the health index and the status of security updates.

> As part of Red{nbsp}Hat's policy, we will provide a support policy and maintenance for any third-party components we depend on that go to end-of-life.

Reviewer: ```suggestion
As part of Red{nbsp}Hat's policy, a support policy and maintenance plan is issued for any third-party components we depend on that go to end-of-life.
```
